cost-analyzer/templates/budget-controller-template.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.budgetController -}}
+{{- if .Values.budgetController.enabled -}}
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: deployment-validation
```

Comment:
This is alright -- maybe prepend the Helm release name like we do for other things? Totally nonblocking idea.

Reply:
there are some interesting rules and naming conventions that need to be hardcoded for the ssl cert to automagically work since it's generated based on a URL. let me get back to this in a v2